### PR TITLE
MISC- Add functionality for hours, minutes, seconds

### PR DESCRIPTION
**Pull request by ekkross**

### CODE REVIEW

File: lib/remi/testing/business_rules.rb [hunk 1]
```diff
@@ -29,11 +29,12 @@ def base_regex
 
     def formulas
       @formulas ||= RegexSieve.new({
-        /\*now(|:[^*]+)\*/i => [:time_reference, :match_now],
-        /\*(today|yesterday|tomorrow)(|:[^*]+)\*/i => [:date_reference, :match_single_day],
-        /\*(this|last|previous|next) (day|month|year|week)(|:[^*]+)\*/i => [:date_reference, :match_single_unit],
-        /\*(\d+)\s(day|days|month|months|year|years|week|weeks) (ago|from now)(|:[^*]+)\*/i => [:date_reference, :match_multiple]
-      })
+                                     /\*now(|:[^*]+)\*/i => [:time_reference, :match_now],
```
Comment (gnilrets) [inline lib/remi/testing/business_rules.rb]: Fix indentation.

File: lib/remi/testing/business_rules.rb [hunk 2]
```diff
@@ -29,11 +29,12 @@ def base_regex
...
+                                     /\*(\d+)\s(hour|hours|minutes|seconds) (ago|from now)(|:[^*]+)\*/i => [:time_reference, :match_time],
```
Comment (gnilrets) [inline lib/remi/testing/business_rules.rb]: Add support for minute and second.

File: lib/remi/testing/business_rules.rb [hunk 3]
```diff
@@ -481,10 +500,10 @@ def values
 
     def value=(arg)
       typed_arg = if metadata[:type] == :json
-        JSON.parse(arg)
-      else
-        arg
-      end
+                    JSON.parse(arg)
```
Comment (gnilrets) [inline lib/remi/testing/business_rules.rb]: Fix indentation

File: lib/remi/testing/business_rules.rb [hunk 4]
```diff
@@ -524,10 +543,10 @@ def initialize(table)
     def parse_formula(value)
       parsed_value = ParseFormula.parse(value)
       case parsed_value
-      when '\nil'
-        nil
-      else
-        parsed_value
+        when '\nil'
```
Comment (gnilrets) [inline lib/remi/testing/business_rules.rb]: Indentation here too.

**Comment by ekkross** [comment]
@gnilrets i think i got all the things in you were looking for!

**Comment by gnilrets** [comment]
👍

**Pull request merged**